octavia-cli/octavia_cli/list/formatting.py

```diff
@@ -0,0 +1,54 @@
+#
+# Copyright (c) 2021 Airbyte, Inc., all rights reserved.
+#
+
+from typing import List
+
+
+def compute_col_width(data: List[List[str]], padding: int = 2) -> int:
+    """Compute column width for display purposes:
+    Find largest column size, add a padding of two characters.
```

Comment:
Since this applies the largest column width to every column in the final table, this means that even columns which are narrow end up having a lot of padding, e.g. the first two columns in the screenshot in the PR description.

It may look a bit better if you rewrite `display_as_table` to calculate the max width of each column _individually_ and apply the corresponding width+padding to each value in the column. This way you can have something that looks like
```
NAME       SOURCE NAME   SOURCE ID
Source 1   Github        0689a244-8467-45d0-8ed0-8118b33f1122
Source 2   Postgres      f631ee3c-f326-4831-b7f1-486e68128daf
```
rather than
```
NAME                        SOURCE NAME                 SOURCE ID
Source 1                    Github                      0689a244-8467-45d0-8ed0-8118b33f1122
Source 2                    Postgres                    f631ee3c-f326-4831-b7f1-486e68128daf
```

Reply:
Thank you for the suggestion, I implemented this in https://github.com/airbytehq/airbyte/pull/9642/commits/9533e931739b93da3494ad78492aa50d4abc5bc3